src/Image/Image.php

```diff
@@ -388,7 +388,7 @@ public function save(?string $path = null, string $type = '', int $quality = 75)
                     \exec($command, $outputArray, $returnCode);
 
                     if ($returnCode !== 0) {
-                        throw new Exception('Image conversion failed');
+                        throw new Exception("Image conversion failed with status {$returnCode}: ".implode("\n", $outputArray));
```

Comment:
Output array might be empty if the command outputs stderr only rather than stdout. Let's redirect stderr to stdout so we capture error messages.